packaging/language/composer.py

```diff
@@ -116,6 +132,8 @@ def parse_out(string):
     return re.sub("\s+", " ", string).strip()
 
 def has_changed(string):
+    if string == "":
```

Comment:
After deeper investigation i see only 'composer init' for existing project returns empty string. So this check mainly for command 'init'.

I can remove this check if You see something bad.